packages/link-bins/test/index.ts

```diff
@@ -225,3 +228,41 @@ test('linkBinsOfPackages() resolves conflicts. Prefer packages that use their na
 
   t.end()
 })
+
+test('linkBins() would throw error if package has no name field', async (t) => {
+  const binTarget = tempy.directory()
+  t.comment(`linking bins to ${binTarget}`)
+  const warn = sinon.spy()
+
+  try {
+    await linkBins(path.join(noNameFixture, 'node_modules'), binTarget, {
+      allowExoticManifests: true,
+      warn,
+    })
+    t.fail('linkBins should fail when package has no name')
+  } catch (err) {
+    t.equal(err.message, normalizePath(`Package in ${path.join(noNameFixture, 'node_modules/simple')} must have a name to get bin linked.`))
```

Comment:
Only the path should be passed to normalizePath(), no the whole error message. Also, this won't work because the path is formed when the error is thrown and normalizing is converts path to linux-style path. So on Windows the path in the error will have `\`. You could use normalize path when adding the path to the error message